codegen/tests/ui/next/missing-endpoint-param.stderr

```diff
@@ -0,0 +1,32 @@
+error: no route provided
+
+         = help: Add a route to your endpoint: `#[shuttle_codegen::endpoint(method = get, route = "/hello")]`
```

Comment:
Maybe this help should only be the `route = "/hello"` part... same for the below errors